job_definitions/notify_suppliers_of_brief_withdrawals.yml

```diff
@@ -37,16 +37,16 @@
       - shell: |
 
           if [ -n "$WITHDRAWN_DATE" ]; then
-            FLAGS="--withdrawn_date=$WITHDRAWN_DATE"
+            PARAMS="$WITHDRAWN_DATE"
           fi
 
           if [ -n "$BRIEF_ID" ]; then
-            FLAGS="$FLAGS --brief_id=$BRIEF_ID"
+            PARAMS="$PARAMS $BRIEF_ID"
           fi
 
           if [ "$DRY_RUN" = "true" ]; then
             FLAGS="$FLAGS --dry-run"
           fi
 
-          docker run digitalmarketplace/scripts scripts/notify-suppliers-of-brief-withdrawal.py '{{ environment }}' "$DM_DATA_API_TOKEN_{{ environment|upper }}" $NOTIFY_API_TOKEN 3e43a131-8399-4674-89d3-b4f80d2e07d4 $FLAGS
+          docker run digitalmarketplace/scripts scripts/notify-suppliers-of-brief-withdrawal.py '{{ environment }}' "$DM_DATA_API_TOKEN_{{ environment|upper }}" $NOTIFY_API_TOKEN 3e43a131-8399-4674-89d3-b4f80d2e07d4 $PARAMS $FLAGS
```

Comment:
Can we define the notify template in predefined-parameters above, rather than inline here?